composer.json

```diff
@@ -19,7 +19,7 @@
         "php": "^7.1.3",
         "ext-xml": "*",
         "doctrine/event-manager": "~1.0",
-        "doctrine/persistence": "~1.0",
+        "doctrine/persistence": "^1.3",
```

Comment:
This should be `^1.3 || ^2.0` - we're looking for absolute parity between the two versions.

Reply:
that would prevent testing with 1.3 for now
OK in a later PR